modules/tinymce/src/core/test/ts/browser/keyboard/KeyboardEventTest.ts

```diff
@@ -0,0 +1,26 @@
+import { Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyContentActions, TinyHooks } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.keyboard.KeyboardEventTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce'
+  }, []);
+
+  it('TINY-10263: getModifierState exists and does not crash the editor', () => {
```

Comment:
Does this actually fail if you remove the fix? I thought the issue was only in certain frameworks like React.

Please delete the test if it can't be made to fail in automated testing, that's an actively harmful test to have.

Reply:
Yes, this does fail if I remove the fix. I checked it before and I checked it again right now.
If you check the issue there's a reproduction that just uses a plain fiddle too. It might have been that it's particularly prominent in react for some reason?

Reply:
makes sense, all good 👍